protos/feast/core/CoreService.proto

```diff
@@ -52,8 +52,11 @@ service CoreService {
     // Create or update and existing feature set.
     //
     // This function is idempotent - it will not create a new feature set if schema does not change.
-    // If an existing feature set is updated, core will advance the version number, which will be
-    // returned in response.
+    // Schema changes will update the feature set if the changes are valid.
+    // All changes except the following are valid:
+    // - Changes to feature set id (name, project)
+    // - Changes to entities
+    // - Changes to feature type
```

Comment:
Feature name changes?

Reply:
Moot, because a feature with a different name is a different feature

Reply:
We know that, but do our users? If you have a `Feature()` and you change the name and apply, you might expect it to rename the feature.

Reply:
Ok, updated.